packages/vite/src/node/plugins/importAnalysis.ts

```diff
@@ -482,7 +485,35 @@ export function importAnalysisPlugin(config: ResolvedConfig): Plugin {
           const isDynamicImport = dynamicIndex > -1
 
           // strip import attributes as we can process them ourselves
+          let importAttributes: Record<string, any> | undefined = undefined
+
           if (!isDynamicImport && attributeIndex > -1) {
+            const raw = source.substring(attributeIndex, expEnd)
+            importAttributes = {}
+
+            // Import attributes are a JavaScript object { foo: 'bar' }
+            // So use Acorn to parse them as such
+            const program = parseJS(`const attributes = ${raw}`, {
```

Comment:
We could also use the `evalValue` utility to simplify this.

Reply:
Ah ok, I didn't know about that one, I'll try the change.

Reply:
I made that change, thanks for the suggestion.